tasking/pulp/tasking/scheduler.py

```diff
@@ -258,7 +258,7 @@ def tick(self):
                           % {'celerybeat_name': self.celerybeat_name})
             ret = self.call_tick(self, self.celerybeat_name)
         except TaskLock.DoesNotExist:
-            TaskLock.objects.get(name=self.celerybeat_name, timestamp__lte=old_timestamp).delete()
+            TaskLock.objects.filter(name=self.celerybeat_name, timestamp__lte=old_timestamp).delete()
```

Comment:
My understanding is that `lock` field indicates that it is a celerybeat lock and it is the important one. The [`name` field](https://github.com/goosemania/pulp/blob/b8abcc929cc3cc4d0a0c2a531d89f2d17480d24f/app/pulp/app/models/task.py#L80) is probably not necessary but it contains the name of the node so it may be useful.